cli/cmd/cluster-apply.go

```diff
@@ -130,7 +129,7 @@ func verifyCluster(kubeconfigPath string, expectedNodes int) error {
 		return errors.Wrapf(err, "failed to set up clientset")
 	}
 
-	cluster, err := lokomotive.NewCluster(client, expectedNodes)
+	cluster, err := k8sutil.NewCluster(client, expectedNodes)
```

Comment:
To be honest, I wouldn't touch `lokomotive` package for now, as it's functionality should go into `Cluster` object, we currently don't have, as this object should wrap all cluster operations.

Also `NewCluster` method should be removed, as it has no value. It never returns error.